next.config.js

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+  images: {
+    domains: ['avatars.githubusercontent.com'],
```

Comment:
What generates this image URL? I'm not seeing it in this patch -- is it a 3p lib?

(Also, FYI https://nextjs.org/docs/app/api-reference/components/image#domains recommends using `remotePatterns` instead, but I'm not sure it buys us anything here, because we can't be more restrictive with the path.)

Reply:
This image url was coming from GitHub's user api which I was using to render the pfp of whoever logged in. I was going to switch to remotePatterns but then realized I've stopped showing the pfp altogether, so I just reverted the changes to this file altogether lol